dev/Installer/src/ProjectReunionInstall.vcxproj

```diff
@@ -0,0 +1,157 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This might be a problem in the future.  Is there a way to define the package location in a MACRO instead of using ../?  I know that packages can be a pain because people might store their packages in a different location.